plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -102,13 +119,22 @@ export type SecureTemplateRenderer = (
 
 export class SecureTemplater {
   static async loadRenderer(options: SecureTemplaterOptions = {}) {
-    const { parseRepoUrl, cookiecutterCompat } = options;
-    let sandbox = undefined;
+    const { parseRepoUrl, cookiecutterCompat, additionalFilters } = options;
+    const sandbox: Record<string, any> = {};
 
     if (parseRepoUrl) {
-      sandbox = {
-        parseRepoUrl: (url: string) => JSON.stringify(parseRepoUrl(url)),
-      };
+      sandbox.parseRepoUrl = (url: string) => JSON.stringify(parseRepoUrl(url));
+    }
+
+    if (additionalFilters) {
+      sandbox.additionalFilters = Object.entries(additionalFilters)
+        .filter(([_, filterFunction]) => !!filterFunction)
+        .reduce((safeFilters, [filterName, filterFunction]) => {
```

Comment:
Actually, there's a very perfectly good reason for why I chose `.reduce` over `Object.fromEntries`...

And it's because I had no idea `Object.fromEntries` existed :smile: 
Thanks, I'll make the change.